### PR TITLE
Added homework 13

### PR DESCRIPTION
**Pull request by tatyana-akulich**

### CODE REVIEW

File: by.teachmeskills/homework13/src/main/java/SalaryTax.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+//        Вычислить сумму налога по прогрессивной шкале в зависимости от полученного заработка:
+//        13% на доход до 10 000, 20% на доход от 10 000 до 50 000, 30% на доход выше 50 000.
+//        Метод должен принимать сумму оплаты труда, а возвращать сумму налога.
+
+class SalaryTax {
```
Comment (testershmester) [inline by.teachmeskills/homework13/src/main/java/SalaryTax.java]: image

Comment (testershmester) [reply by.teachmeskills/homework13/src/main/java/SalaryTax.java]: package is not defined

File: by.teachmeskills/homework13/src/test/java/SalaryTaxTest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class SalaryTaxTest {
+    private double salary;
+
+    @Test
+    public void testSalaryPositiveLess10000() {
```
Comment (testershmester) [inline by.teachmeskills/homework13/src/test/java/SalaryTaxTest.java]: data provider can be considered as improvement

File: by.teachmeskills/homework13/src/main/java/SalaryTax.java (hunk 1 above)
Comment (testershmester) [inline by.teachmeskills/homework13/src/main/java/SalaryTax.java]: public?

**Review by testershmester — APPROVED** [review]
The code is OK, but I have troubles with running your code. I didn't manage to build the project.

File: by.teachmeskills/homework13/src/main/java/SalaryTax.java (hunk 1 above)
Comment (tatyana-akulich) [reply by.teachmeskills/homework13/src/main/java/SalaryTax.java]: I've fixed the structure and added Dataprovider

**Pull request merged**